### DISCUSSION TITLE
Liking plugin update notifications

### DISCUSSION THREAD

**Discussion by mikethebee** [grafana/grafana-csv-datasource#22] [Show and tell]
Just to say I am liking the CSV plugin update notification. I am using it on RaspberryPi OS Buster to display a home sensor dashboard for my TheThingsNetwork connected sensors that use mqtt and awk on another server to parse and format the data. Thank you for the continuing to work on improvements.

**Comment by marcusolsson** [comment]
Thanks for letting me know! So you basically have a script that subscribe to MQTT messages and append them to a CSV file on disk? Really cool!

Thanks for trying it out and for helping improve it! ❤️

**Reply by mikethebee** [reply]
Yes, [TheThingsNetwork](http://thethingsnetwork.org) provide free to use global community servers that host an mqtt publisher which forwards any messages received from the your lpwan sensors via a network of wireless gateways (similar to wifi access points, but with range of several miles/km). Ipwan is also low power so a couple of AA cells last months and the longer range is ideal for big gardens with sensors and actuators (eg. waterbutt irrigation, weather-station, inside a fridge)

The published data is in a non-standard json format so I use awk to sort the time format and change the json to csv appropriate for the panel widget expectations. I'm still a noob in grafana and influxdb so starting with flat csv files allows me to work on the data flow before trying to get clever with db and widgets.

![image](https://user-images.githubusercontent.com/16389485/105201196-8013a780-5b38-11eb-88fc-e5dfc290b8d6.png)
